codeSnippets/snippets/client-call-id/README.md

```diff
@@ -0,0 +1,27 @@
+# Client call id
+
+A sample Ktor project showing how to use the [CallId](https://ktor.io/docs/client-call-id.html) client plugin.
+The project consists of two parts: an API service and a client.
+The API service utilizes the `callId` plugin on the server and configures it to retrieve a call ID from the request
+header or use a generated call ID.
+
+> This sample is a part of the [codeSnippets](../../README.md) Gradle project.
+
+## Running
+
+1. To start the API service, navigate to `CallIdService.kt` and click on the play button next to the `main()` function.
+
+2. Run the client sample by executing the following command:
+
+```bash
+./gradlew :client-call-id:run
+```
+
+The client makes 3 requests to the API and prints out information about the call ID.
+
+The expected output is:
+```
+in call: call-id-client, in header: call-id-client, in query param: call-id-client
```

Comment:
Could we print it in a way that indicates how the data flow goes? For instance, in a server client we can use call id from the request as a part, so we can track the whole chain of the calls